tools/ovc/openvino/tools/ovc/moc_frontend/pipeline.py

```diff
@@ -109,10 +119,17 @@ def add_names_to_tensors(model: InputModel, places: List[Place]):
     model_inputs = input_model.get_inputs()
     inputs_equal = True
     if user_shapes:
+
+        # TODO: Remove this line when new 'cut' helper is introduced
+        raise_exception_for_input_output_cut(model_inputs, user_shapes, True)
```

Comment:
Are we sure that `model_inputs` is correct for all FEs (even for TF FE for different formats)?